src/Compilers/CSharp/Portable/Symbols/FunctionPointers/FunctionPointerMethodSymbol.cs

```diff
@@ -714,8 +766,7 @@ internal int GetHashCodeNoParameters()
             DiagnosticInfo? info = null;
             CalculateUseSiteDiagnostic(ref info);
 
-            if (CallingConvention.IsCallingConvention(CallingConvention.ExtraArguments) ||
-                CallingConvention.IsCallingConvention(CallingConvention.FastCall))
```

Comment:
Why was `FastCall` removed? Do we support this?

Reply:
We do, though the only place that you could use it is on Mono on Windows x86.

---
In reply to: [467364831](https://github.com/dotnet/roslyn/pull/46467#discussion_r467364831) [](ancestors = 467364831)